apps/arena/lib/arena/game_updater.ex

```diff
@@ -874,6 +876,24 @@ defmodule Arena.GameUpdater do
     end)
   end
 
+  defp initialize_pools(pools, last_id) do
+    Enum.reduce(pools, {Map.new(), last_id}, fn pool, {pools_acc, last_id} ->
+      last_id = last_id + 1
+
+      pools_acc =
+        Map.put(
+          pools_acc,
+          last_id,
+          Entities.new_pool(
+            pool
+            |> Map.merge(%{id: last_id, owner_id: 9999, skill_key: "0", status: :READY})
```

Comment:
Shouldn't they have their initial status in the config?

Reply:
nope, these pools are environmental ones, I don't see why they should start as waiting and be activated later, they should always be active

Reply:
Great!